drivers/spi/Kconfig.xec_qmspi_v2

```diff
@@ -0,0 +1,18 @@
+# Microchip XEC QMSPI FULL DUPLEX, QMSPI LDMA
+
+# Copyright (c) 2022 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+choice
+	prompt "Select Microchip XEC QMSPI driver"
+	default SPI_XEC_QMSPI_LDMA
+	depends on SPI && SOC_SERIES_MEC172X
```

Comment:
`SPI` here should be redundant since this is already under drivers/spi. `SOC_SERIES_MEC172X` could also probably go and instead I think you could use something like

`depends on DT_HAS_MICROCHIP_XEC_QMSPI_FULL_DUPLEX || DT_HAS_MICROCHIP_XEC_QMSPI_LDMA`

Reply:
updated to
`depends on DT_HAS_MICROCHIP_XEC_QMSPI_FULL_DUPLEX_ENABLED || DT_HAS_MICROCHIP_XEC_QMSPI_LDMA_ENABLED`